packages/experimental/Avatar/src/JSAvatar.styling.ts

```diff
@@ -68,36 +81,33 @@ export const stylingSettings: UseStylingOptions<JSAvatarProps, AvatarSlotProps,
           },
         };
       },
-      ['coinColorFluent', 'backgroundColor', 'physicalSize', ...borderStyles.keys],
+      ['coinColorFluent', 'backgroundColor', 'width', 'height', ...borderStyles.keys],
     ),
     image: buildProps(
       (tokens: JSAvatarTokens) => {
-        const { physicalSize } = calculateEffectiveSizes(tokens);
-
         return {
           style: {
             borderRadius: tokens.borderRadius,
-            width: physicalSize,
-            height: physicalSize,
+            width: tokens.width,
+            height: tokens.height,
           },
         };
       },
-      ['physicalSize', 'borderRadius'],
+      ['borderRadius', 'width', 'height'],
     ),
     icon: buildProps(
       (tokens: JSAvatarTokens, theme: Theme) => {
-        const { iconSize, iconStrokeWidth } = calculateEffectiveSizes(tokens);
-        const iconSizeAdjusted = iconSize + iconStrokeWidth * 2;
+        const iconSizeAdjusted = tokens.iconSize + tokens.iconStrokeWidth * 2;
         const iconStrokeColor = tokens.iconStrokeColor || theme.colors.background;
         return {
           style: {
             position: 'absolute',
             width: iconSizeAdjusted,
             height: iconSizeAdjusted,
-            bottom: -iconStrokeWidth,
-            end: -iconStrokeWidth,
-            borderRadius: iconSizeAdjusted / 2,
-            borderWidth: iconStrokeWidth,
+            bottom: -tokens.iconStrokeWidth,
+            end: -tokens.iconStrokeWidth,
+            borderRadius: tokens.iconSize / 2,
```

Comment:
Is there a reason this changed from iconSizeAdjusted to tokens.iconSIze?

Reply:
I moved iconSize to tokens. Icon size from helpers was also from an old code of PersonaCoin. Maybe we won't need this file at all when I finish with sizes and colors.

Reply:
I removed some styles from icon as it's not used. For example, iconStroke was used for activity-ring, but currently activeAppearance is used for that